### DISCUSSION TITLE
More consistend config logs viewer

### DISCUSSION THREAD

**Discussion by wendevlin** [home-assistant/frontend#20845] [Configuration]
Hello,

I am currently working on an HA Add-on, so I have to check the supervisor logs from time to time.
Then I figured out some points I want to discuss and improve:

1. Home assistant core logs are sorted DESC and all other log categories (eg. Supervisor) are sorted ASC. It would be nice if this is consistent for all logs or there is some kind of indicator that the user knows how it is sorted.
2. the refresh button is always placed on the top, this leads to a lot of scrolling for all logs except the home assistant core:
  - Scroll down to see the latest updates
  - scroll up again to refresh
  - then scroll down again to see the new updates
4. There is a download full log button beside the refresh button expect in the HA core logs (there it is a button on the bottom which looks different with text in it)

I gave this into one issue because it is related, I would like to help but first get into a discussion how to solve it. I think it is a good first issue for me to contribute. 

![2024-05-21_20-28](https://github.com/home-assistant/frontend/assets/12148533/27735b82-4b5f-4911-97d1-a3d9c8a3fbed)
![2024-05-21_20-27](https://github.com/home-assistant/frontend/assets/12148533/2755b2f2-3ba5-4104-a884-4d88baa6b5df)
![2024-05-21_20-29](https://github.com/home-assistant/frontend/assets/12148533/f44b03fb-b146-40ba-9f0d-5839fcd9b36f)

**Comment by ildar170975** [comment]
Regarding the "Refresh" button:
[Log page: make a "Refresh" button sticky, keep scroll position after pressing the "Refresh" button](https://github.com/home-assistant/frontend/discussions/15402)

**Reply by ildar170975** [reply]
In 2024.11 a "live log" was added, the "Refresh" button is placed on a fixed panel:
![385614238-90a1b781-2eca-49b7-9cf6-9f848b3ad2ce](https://github.com/user-attachments/assets/fa9052cc-9f41-4c6c-91cd-00392761b54c)
